package.xml

```diff
@@ -1,18 +1,22 @@
-<package>
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>geometric_shapes</name>
-  <version>0.6.1</version>
+  <version>0.6.2</version>
```

Comment:
is it necessary to bump the version here? my understanding is that only bloom does this during the release process @130s ?

Reply:
It's ok to modify version number along with other code change although that sounds unusual.  IMO it's typically done during release flow, e.g. in Catkin's world, `catkin_prepare_release` nicely takes care of it automatically. Using a toolchain like that is preferable as it allows release flow to be more systematic and organized.

And the change suggested in this PR seems non-trivial, so bumping minor or even major number makes more sense to me (if the repo follows common major-minor-patch version semantics).

Reply:
We should keep ROS and ROS2 versions separate. In the RQT port we changed all of the ROS2 versions to 1.0.0 although I am not sure that is ideal